src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -86,11 +86,22 @@ public class AsymmetricSignatureProvider : SignatureProvider
             { SecurityAlgorithms.RsaSha512Signature, 1024 }
         };
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AsymmetricSignatureProvider"/> class used to create and verify signatures.
+        /// </summary>
+        /// <param name="key">That will be used for cryptographic operations.</param>
```

Comment:
Perhaps something like ....:  "the <see cref="SecurityKey"> that will be used for signature operations ...."